Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/EVM/EvmCalls.cs

```diff
@@ -0,0 +1,277 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Transactions;
+using ChainSafe.Gaming.MultiCall;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Contracts.QueryHandlers.MultiCall;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Scripts.EVM.Token;
+using UnityEngine;
+using Web3Unity.Scripts.Prefabs;
+using ABI = ChainSafe.Gaming.UnityPackage.ABI;
+
+public class EvmCalls : MonoBehaviour
+{
+    #region Fields
+    
+    #region IPFS
+
+    private string apiKey = "YOUR_CHAINSAFE_STORE_API_KEY";
+    private string data = "YOUR_DATA";
+    private string bucketId = "BUCKET_ID";
+    private string path = "/PATH";
+    private string filename = "FILENAME.EXT";
+
+    #endregion
+    
+    # region Contract Send
+        
+    private string methodSend = "addTotal";
+    private string abiSend = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
+    private string contractAddressSend = "0x7286Cf0F6E80014ea75Dbc25F545A3be90F4904F";
+    private int increaseAmountSend = 1;
+        
+    #endregion
+
+    #region Contract Call
+
+    private string methodCall = "myTotal";
+    private string abiCall = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
```

Comment:
This abi is the same as the one on line 33

Reply:
@kantagara @robGG1997 So just wanna check if this is alright, the code here, @sneakzttv says is a template for using our examples so would be an exception to the rule when avoiding duplication of variables. Does that make sense?

Reply:
same as below

Reply:
We need to have a single point of reference as much as possible unless our use case explicitly demands we use different ones